tests/core.js

```diff
@@ -319,14 +319,16 @@ require([
         });
 
         it("should add question after sole hidden value", function (done) {
-            util.init({core: {onReady: function () {
-                util.addQuestion("DataBindOnly", "hidden");
-                util.addQuestion.bind({prevId: "hidden"})("Text", "text");
-                util.assertJSTreeState(
-                    "hidden",
-                    "text"
-                );
-                done();
+            util.init({
```

Comment:
Changing this test to using the `util.loadXML`  pattern makes another test fail. (but only when you run 'Vellum core tests', you can run the failing test by itself and it will pass)